### ISSUE TITLE
Error in the pull function: Sync failed

### ISSUE THREAD

**Issue by poetaman** [coddingtonbear/inthe.am#402]
It used to work, but in last months not really. This is what I get:

```
❯ task sync
Syncing with inthe.am:53589

Failed to receive header: Error in the pull function.
Sync failed.  Could not connect to the Taskserver.
```

**Comment by wpawlikowski** [comment]
Up. Doesn't work since yesterday. Same thing with inthe.am.

**Comment by poetaman** [comment]
@wpawlikowski Actually it works for me now since October. Don't remember what all I changed, I think ``taskd.credentials`` had changed  https://inthe.am/configure/synchronization change over time?  @coddingtonbear Do ``taskd.credentials`` expire after a period? Not sure as I backed up the local files in a different folder and had reinstalled everything. One other thing I changed is that all of my directories are all my files are under ``~/.task``, and the config is in ``~/.taskrc``. I had tried other combination of folder and config file locations, but they didn't work.

Contents of ``~/.taskrc``:
```
data.location=/Users/poetaman/.task
taskd.certificate=/Users/poetaman/.task/private.cert
taskd.key=/Users/poetaman/.task/private.key
taskd.ca=/Users/poetaman/.task/ca.cert.pem
taskd.server=inthe.am:<somenumber>
taskd.credentials=inthe_am/<somename>/<someid>
taskd.trust=ignore hostname
```
